libindy/src/services/wallet/wallet.rs

```diff
@@ -239,6 +252,12 @@ impl Wallet {
         Ok(())
     }
 
+    pub(super) fn rotate_key(&self, new_master_key: &[u8]) -> Result<(), WalletError> {
+        let new_keys = self.keys.encrypt(new_master_key);
```

Comment:
`new_keys` is misleading here, please rename it. Actually keys are not changed, only encrypted representation are changed.

Reply:
yes, new_keys might be misleading. I agree